### DISCUSSION TITLE
javascript help

### DISCUSSION THREAD

**Discussion by max69300** [community/community#142828] [Programming Help]
### Body

 hello everyone in JavaScript I have a problem I can't click twice in a row in my function its click only once thank you for your answers

function click2()
 {
 document.getElementById("buttonenregistre").click();
 document.getElementById("buttonenregistre").click();
 };

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by fahmifareed** [comment]
Here's a way to make your button click twice in a row with some tweaks to ensure it works smoothly:

### **Updated Code:**

```javascript
function click2() {
  const btn = document.getElementById("buttonenregistre");
  
  // Simulate first click
  btn.click();
  
  // Use a brief delay before the second click
  setTimeout(() => {
    btn.click();
  }, 50); // 50ms delay, tweak as needed
}
```

### **Why This Works:**
- The `setTimeout` adds a short delay before the second click, ensuring the browser has time to handle the first one.
- Using a lower delay (like 50ms) might make the clicks feel more responsive.

**Comment by max69300** [comment]
goog phank you

**Reply by fahmifareed** [reply]
@max69300 you welcome brother
